shell/platform/embedder/embedder.h

```diff
@@ -1118,9 +1121,82 @@ typedef struct {
   const char* tooltip;
 } FlutterSemanticsNode;
 
+/// A node that represents some semantic data.
```

Comment:
```suggestion
/// A node in the Flutter semantics tree.
```

Yeah, it's still mostly tautological, but it does point out that this is the same thing as a `SemanticsNode` in the framework, and it's a couple characters shorter.

Reply:
Updated!